designer/client/modal.js

```diff
@@ -11,7 +11,10 @@ function Modal(props) {
         <a
           title="Close"
           className="close govuk-body govuk-!-font-size-16"
-          onClick={(e) => props.onHide(e)}
+          onClick={(e) => {
+            e.preventDefault();
+            props.onHide(e);
+          }}
```

Comment:
it might be neater to do 
```
          onClick={props.onHide}
```
`e(vent)` will be passed in, so you can do the e.preventDefault in the onHide method. no need to action, your discretion

Reply:
just to be sure those using it will not forget to call preventDefault - i think it is better to leave it